python/samples/03.adaptiveCards.a.typeAheadBot/README.md

```diff
@@ -0,0 +1,53 @@
+# Teams Type-Ahead Bot
+
+This sample shows how to incorporate Adaptive Cards into a Microsoft Teams application using [Bot Framework](https://dev.botframework.com) and the Teams AI SDK. Type-Ahead bot gives an enhanced search experience with Adaptive Cards to search and select data.
+
+<!-- @import "[TOC]" {cmd="toc" depthFrom=1 depthTo=6 orderedList=false} -->
+
+<!-- code_chunk_output -->
+
+-   [Teams Type-Ahead Bot](#teams-type-ahead-bot)
+    - [Interacting with the bot](#interacting-with-the-bot)
+    - [Setting up the sample](#setting-up-the-sample)
+    - [Testing the sample](#testing-the-sample)
+        - [Using Teams Toolkit for Visual Studio Code](#using-teams-toolkit-for-visual-studio-code)
+
+<!-- /code_chunk_output -->
+
+## Interacting with the bot
+
+You can interact with this bot by sending it a message. Send it a message of `static` or `dynamic` and it will respond with an Adaptive Card. Selecting an option and submitting will send a message to the bot with the selected option(s).
+
+![screenshot](./assets/screenshot.PNG)
+
+## Setting up the sample
+
+1. Clone the repository
+
+    ```bash
+    git clone https://github.com/Microsoft/teams-ai.git
+    ```
+
+## Testing the sample
+
+The easiest and fastest way to get up and running is with Teams Toolkit as your development guide. To use Teams Toolkit to automate setup and debugging, please [continue below](#using-teams-toolkit-for-visual-studio-code).
```

Comment:
Include directions above here about running install/build?

Reply:
That happens through the Python extension! (steps 3 and 8 in "Using Teams Toolkit for Visual Studio Code")